assets/src/settings-page/settings-footer.js

```diff
@@ -28,13 +101,27 @@ export function SettingsFooter() {
 		( 'reader' === themeSupport && ! readerTheme );
 
 	return (
-		<section className="settings-footer">
-			<Button isPrimary disabled={ disabled } isBusy={ savingOptions || downloadingTheme } type="submit">
-				{ __( 'Save changes', 'amp' ) }
-				<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 64 64">
-					<path d="M43.16 10.18c-0.881-0.881-2.322-0.881-3.203 0s-0.881 2.322 0 3.203l16.335 16.335h-54.051c-1.281 0-2.242 1.041-2.242 2.242 0 1.281 0.961 2.322 2.242 2.322h54.051l-16.415 16.335c-0.881 0.881-0.881 2.322 0 3.203s2.322 0.881 3.203 0l20.259-20.259c0.881-0.881 0.881-2.322 0-3.203l-20.179-20.179z" />
-				</svg>
-			</Button>
+		<section className="amp-settings-nav">
+			<div className="amp-settings-nav__inner">
+				<Button isPrimary disabled={ disabled } isBusy={ savingOptions || downloadingTheme } type="submit">
+					{ savingOptions || downloadingTheme ? __( 'Saving', 'amp' ) : __( 'Save', 'amp' ) }
+					<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 64 64">
+						<path d="M43.16 10.18c-0.881-0.881-2.322-0.881-3.203 0s-0.881 2.322 0 3.203l16.335 16.335h-54.051c-1.281 0-2.242 1.041-2.242 2.242 0 1.281 0.961 2.322 2.242 2.322h54.051l-16.415 16.335c-0.881 0.881-0.881 2.322 0 3.203s2.322 0.881 3.203 0l20.259-20.259c0.881-0.881 0.881-2.322 0-3.203l-20.179-20.179z" />
+					</svg>
+				</Button>
+				{ error && <ErrorNotice errorMessage={ error.message || __( 'An error occurred. You might be offline or logged out.', 'amp' ) } /> }
+				{ ! error && ! hasOptionsChanges && didSaveOptions && ! downloadingTheme && (
+					<AMPNotice
+						className={ `amp-save-success-notice ${ savedNoticeClass }` }
```

Comment:
Should the notice not be hidden once `savedNoticeClass` is set to `dismissed`?

Reply:
Removed the class entirely in 9d1ec07